packages/calypso-e2e/src/lib/components/editor-block-list-view-component.ts

```diff
@@ -2,8 +2,16 @@ import { Locator, Page } from 'playwright';
 
 const selectors = {
```

Comment:
All of these file changes were originally for a way to take editor actions to delete all. Since it doesn't apply to mobile, I ended up pivoting away. But, the implementation seemed potentially useful for future tests, so I decided to leave in! 

I can definitely pull out if needed 👍

Reply:
I don't mind leaving it in for now. We will likely have to update it when it becomes needed though 😉